### PR TITLE
[CI Environment] Replaced default `namePrefix`

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Replaced default `namePrefix` from `<<` & `>>` to `[[` & `]]`

| Pipeline |
| - |
| [AnalysisServices - Servers](https://github.com/Azure/ResourceModules/actions/workflows/ms.analysisservices.servers.yml) |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1 [hunk 1]
```diff
@@ -76,7 +76,7 @@ function Convert-TokenInFile {
             if ($TokenNameValueObject[$_] -is [System.Security.SecureString]) {
                 $TokenNameValueObject[$_] = $TokenNameValueObject[$_] | ConvertFrom-SecureString -AsPlainText
             }
-            $File = $File -replace $_, $TokenNameValueObject[$_]
+            $File = $File -replace [Regex]::Escape($_), $TokenNameValueObject[$_]
```
Comment (eriqua) [inline utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1:79]: Is this change intended or coming from previous tries to fix?

File: utilities/tools/Set-ModuleReadMe.ps1 [hunk 2]
```diff
@@ -1004,7 +1004,7 @@ function Set-DeploymentExamplesSection {
 
             $rawBicepExampleString = ($rawBicepExample | Out-String)
             $rawBicepExampleString = $rawBicepExampleString -replace '\$\{serviceShort\}', $serviceShort
-            $rawBicepExampleString = $rawBicepExampleString -replace '\$\{namePrefix\}', '<<namePrefix>>' # Replacing with empty to not expose prefix and avoid potential deployment conflicts
+            $rawBicepExampleString = $rawBicepExampleString -replace '\$\{namePrefix\}', '[[namePrefix]]' # Replacing with empty to not expose prefix and avoid potential deployment conflicts
```
Comment (eriqua) [inline utilities/tools/Set-ModuleReadMe.ps1:1007]: Not for this PR but I'm wondering if we have a mismatch between allowing a customized token prefix and suffix in the settings.yml and at the same time hardcoding those in some of the utilities

File: utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1 (hunk 1 above)
Comment (AlexanderSehr) [reply utilities/pipelines/tokensReplacement/helper/Convert-TokenInFile.ps1:79]: It is intended. The `[` is a rege character and `-replace` (other than `.replace()`) uses regex

File: utilities/tools/Set-ModuleReadMe.ps1 (hunk 2 above)
Comment (AlexanderSehr) [reply utilities/tools/Set-ModuleReadMe.ps1:1007]: We do. Good point. We'd need to load the settings YAML here too.

Comment (AlexanderSehr) [reply utilities/tools/Set-ModuleReadMe.ps1:1007]: I can fix it here or open a new bug. Your call

Comment (eriqua) [reply utilities/tools/Set-ModuleReadMe.ps1:1007]: Since not related to this fix specifically and since other PRs are waiting for it let's open a bug and address it there

Comment (AlexanderSehr) [reply utilities/tools/Set-ModuleReadMe.ps1:1007]: Opened bug https://github.com/Azure/ResourceModules/issues/3429

**Review by eriqua — APPROVED** [review]
🆗
